src/legacy/ui/public/utils/mapping_setup.js

```diff
@@ -17,24 +17,23 @@
  * under the License.
  */
 
-import angular from 'angular';
-import _ from 'lodash';
+import { mapValues } from 'lodash';
 
 // eslint-disable-next-line import/no-default-export
-export default function MappingSetupService() {
-  const mappingSetup = this;
+export function mappingSetupService() {
```

Comment:
Instead of returning a function, you can return an object with a method. Therefore you won't need to execute `mappingSetupService` before calling `expandShorthand`

Reply:
:+1: looks like we could even just return the function `expandShorthand`. Since we're planning on converting this to TS in a second pass, I'd suggest we focus this PR on removing angular, and then do some of the refactoring once we have TS in place.